core/src/test/java/org/elasticsearch/search/aggregations/bucket/FiltersIT.java

```diff
@@ -485,4 +487,14 @@ public void testEmptyAggregationWithOtherBucket() throws Exception {
         assertThat(other.getDocCount(), is(0L));
     }
 
+    private static KeyedFilter[] randomOrder(KeyedFilter... filters) {
+        for (int i = 0; i < filters.length; i++) {
+            KeyedFilter tmp = filters[i];
+            int index = randomInt(filters.length - 1);
+            filters[i] = filters[index];
+            filters[index] = tmp;
+        }
+        return filters;
+    }
```

Comment:
if you call `Collections.shuffle(Arrays.asList(filters))``, I believe the array will be modified in place?